### PR TITLE
[DOCS] Editing usage of Arduino as a component in ESP-IDF

### PR DESCRIPTION
**Pull request by P-R-O-C-H-Y**
## Summary
Fixed link to lib-builder.
Divided "usage" to 2 options (in 1 you have to change main.c to main.cpp).
Some styling fixes.

## Impact
None.

### CODE REVIEW

**Comment by VojtechBartoska** [comment]
included in #5839

**Review by VojtechBartoska — APPROVED** [review]

File: docs/source/esp-idf_component.rst [hunk 1]
```diff
@@ -5,15 +5,15 @@ Arduino as a ESP-IDF component
...
 
 Installation
 ------------
 
-.. note:: Latest Arduino Core ESP32 version is now compatible with [ESP-IDF v4.4](https://github.com/espressif/esp-idf/tree/release/v4.4). Please consider this compability when using Arduino as component in ESP-IDF.
+.. note:: Latest Arduino Core ESP32 version is now compatible with `ESP-IDF v4.4 <https://github.com/espressif/esp-idf/tree/release/v4.4>`_. Please consider this compability when using Arduino as component in ESP-IDF.
 
 - Download and install `ESP-IDF <https://github.com/espressif/esp-idf>`_.
-- Create blank idf project (from one of the examples).
+- Create blank idf project (use sample_project from /examples/get-started) or choose one of the examples.
```
Comment (pedrominatel) [inline docs/source/esp-idf_component.rst:16]: idf -> ESP-IDF. Usually, we use ESP-IDF in capital letters as our official naming rule.

File: docs/source/esp-idf_component.rst [hunk 2]
```diff
@@ -45,14 +54,17 @@ Installation
         delay(1000);
     }
 
-- Otherwise, you need to implement ```app_main()``` and call ```initArduino();``` in it.
+Option 2. Using IDF appmain()
```
Comment (pedrominatel) [inline docs/source/esp-idf_component.rst:57]: ESP-IDF app_main()

**Review by pedrominatel — CHANGES_REQUESTED** [review]
PTAL at my considerations.

**Review by pedrominatel — APPROVED** [review]

**Comment by pedrominatel** [comment]
LGTM @P-R-O-C-H-Y. Thank you!

**Pull request merged**